controllers/nodehealthcheck_controller.go

```diff
@@ -75,8 +74,9 @@ const (
 )
 
 var (
-	clusterUpgradeRequeueAfter = 1 * time.Minute
-	currentTime                = func() time.Time { return time.Now() }
+	clusterUpgradeRequeueAfter       = 1 * time.Minute
+	logWhenCRPendingDeletionDuration = 10 * time.Second
```

Comment:
Why did you choose 10 seconds?

Reply:
probably long enough in most cases to let remediators remove the finalizer, but not too long to miss this log in case we want it

Reply:
Gotcha